File: base/relogio/.cache/draft/ts/time.ts
class Time {
    private hour: number;
    private minute: number;
    private second: number;

    constructor(hour: number, minute: number, second: number) {

    }
    setHour(hour: number): void {
    }
    setMinute(minute: number): void {
    }

    setSecond(second: number): void {
    }
    getHour(): number {
    }
    getMinute(): number {
    }

    getSecond(): number {
    }
    nextSecond(): void {
    }
    toString() {
        let p2 = (n: number) => ("" + n).padStart(2, "0");
        return p2(this.hour) + ":" + p2(this.minute) + ":" + p2(this.second);
    }
}

export {Time};

File: base/relogio/.cache/draft/java/Time.java
class Time {
    private int hour   = 0;
    private int minute = 0;
    private int second = 0;

    public Time(int hour, int minute, int second) {
    }

    public void setHour(int hour) {
    }
    public void setMinute(int minute) {
    }
    public void setSecond(int second) {
    }
    public int getHour() {
    }
    public int getMinute() {
    }
    public int getSecond() {
    }
    public void nextSecond() {
    }
    public String toString() {
        return String.format("%02d:%02d:%02d", hour, minute, second);
    }
}